web/viewer.js

```diff
@@ -1061,9 +1062,13 @@ var PDFViewerApplication = {
     });
   },
 
-  setInitialView: function pdfViewSetInitialView(storedHash, scale) {
+  setInitialView: function pdfViewSetInitialView(storedHash, scale, rotation) {
     this.isInitialViewSet = true;
 
+    if (rotation && rotation !== this.pageRotation) {
+      this.rotatePages(rotation);
```

Comment:
According to https://github.com/mozilla/pdf.js/blob/master/web/viewer.js#L1286, the argument is the angle `delta`.
So it seems to me that this line will implicitly assume that `this.pageRotation === 0`. Doesn't it need to be `this.rotatePages(rotation - this.pageRotation)` instead?